doc/src/manual/arrays.md

```diff
@@ -182,7 +182,7 @@ julia> sum(1/n^2 for n=1:1000)
 When writing a generator expression with multiple dimensions inside an argument list, parentheses
 are needed to separate the generator from subsequent arguments:
 
-```julia
+```julia-repl
```

Comment:
There is now: JuliaDocs/Documenter.jl#487 Also, the same applies to the invalid UTF-8 example in `manual/strings.md`.